lisa/operating_system.py

```diff
@@ -362,7 +362,7 @@ def _install_packages(
             )
 
     def _package_exists(self, package: str, signed: bool = True) -> bool:
-        command = f"apt list --installed | grep -Ei {package}"
+        command = f"apt list --installed {package}"
```

Comment:
I did a little more digging on this. I think we can remove grep here but guess we should run this command with option -qq and then if result.stdout is not empty then we can be sure that package exists.

~$ apt -qq list installed nano
nano/focal,now 4.8-1ubuntu1 amd64 [installed]
~$ apt -qq list installed python
~$ apt -qq list installed python3
python3/focal,now 3.8.2-0ubuntu2 amd64 [installed,automatic]

Reply:
Thank you @sharsonia.
I will use dpkg --get-selections | grep -q "^$pkg[[:space:]]*install$".
`apt list --installed {package}` and `apt -qq list installed python` return code is always 0.